src/amo/components/Errors/ErrorPage/index.js

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import makeClassName from 'classnames';
+import NestedStatus from 'react-nested-status';
+import PropTypes from 'prop-types';
+
+import './styles.scss';
+
+export default class ErrorPage extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    code: PropTypes.number,
+    header: PropTypes.node,
+  };
+
+  render() {
+    const { children, className, header, code } = this.props;
+
+    return (
+      <NestedStatus code={code}>
+        <section
```

Comment:
The original files used `Card` inside `NestedStatus`, but this new component uses `section` instead. Is there a reason for this? I anticipate visual regressions as a result of this.

Reply:
I thought I should keep all the css class related elements all together in the basic component. So, now I will be doing - 
```
<ErrorPage code={404}>
      <Card
          className="Errors NotFound"
          header={i18n.gettext('Oops! We can’t find that page')}
        >
        .....
      </Card>
</ErrorPage >

is this okay ?

Reply:
Instead of this, you can move `Card` into the new component that you've created (which you've called `ErrorPage`, but I just realized it should actually be called `Error` or `ErrorComponent`). Then you can just pass the `header` into your new component, and it can pass that `header` into `Card`.

Reply:
`ErrorComponent` sounds good. I was searching for a good name. Thanks!